src/test/scala/com/microsoft/hyperspace/index/IndexManagerTests.scala

```diff
@@ -32,6 +32,8 @@ import com.microsoft.hyperspace.util.FileUtils
 class IndexManagerTests extends SparkFunSuite with SparkInvolvedSuite {
   private val sampleParquetDataLocation = "src/test/resources/sampleparquet"
   private val indexStorageLocation = "src/test/resources/indexLocation"
+  private val absoluteIndexLocation = TestUtils.makeAbsolute(indexStorageLocation)
```

Comment:
Can't we just make `indexStorageLocation` as absolute path instead of introducing a new var?

Reply:
question: would `makeAbsolute` work even if the path doesn't exist?

Reply:
thanks @imback82, done. Yeah it works (e.g. index log manager and index data manager are created before the folders exist)